src/cookies.rs

```diff
@@ -9,7 +9,7 @@ use crate::error;
 /// Type alias for a [cookie::Cookie]
 pub type Cookie<'a> = cookie::Cookie<'a>;
 
-/// Representation of a cookie as [defined by WebDriver](https://www.w3.org/TR/webdriver1/#cookies).
+/// Representation of a cookie as [defined by WebDriver](https://w3.org/TR/webdriver1/#cookies).
```

Comment:
Why remove all the `www`s? That's the canonical URL that ends up being redirected to to after all?

Reply:
It also causes a _lot_ of noise in this PR.

Reply:
I do tend keep links as small as possible, because long ones make problem for `rustfmt`. I'll separate this change into another small PR, if the noise bothers.

Reply:
Yeah, I like to keep PRs focused, otherwise reviewing gets much harder. And tracking through git history becomes more of a pain :sweat_smile: Thanks!